packages/devextreme/js/__internal/ui/chat/chat.ts

```diff
@@ -43,6 +43,7 @@ class Chat extends Widget<Properties> {
       dataSource: null,
       user: { id: new Guid().toString() },
       onMessageSend: undefined,
+      showDateHeaders: true,
```

Comment:
let's update the naming

```suggestion
      showDayHeaders: true,
```

Reply:
renamed